src/css/main.less

```diff
@@ -222,6 +222,112 @@ a,
 }
 
 /* ==============
+ * Icons
```

Comment:
How about adding a comment saying taken from Canvas with a link to the github repo?
I think it feels bad to be "taking" without credit...